ui/app/components/ui/popover/index.scss

```diff
@@ -107,4 +110,16 @@
       margin: 0 auto;
     }
   }
+
+  &-arrow {
+    width: 22px;
+    height: 22px;
+    background: white;
+    position: absolute;
+    transform: rotate(45deg);
+    top: -6px;
```

Comment:
Nit: These positions don't make sense for the general `Popover` component; they're specific to this specific case.

Could you extract some or all of these rules into a `home__` class instead? One specific to this instance.

Reply:
addressed in 9bfe721f8